### DISCUSSION TITLE
archives/bj-pfd2/pfd-tutor/

### DISCUSSION THREAD

**Discussion by giscus** [songtianlun/songtianlun.github.io#13] [Announcements]
# archives/bj-pfd2/pfd-tutor/

BJ-PFD2 文档清单 BJ-PFD2 - 入口 BJ-PFD2 - 快速开始 BJ-PFD2 - BJ 使用方法 BJ-PFD2 - PFD 使用方法 本节介绍 BJ-PFD 中个人看板（PFD）的使用方法，使用前请确保您已将 Bullet Journel 克隆到您个人的 Notion 中

https://www.frytea.com/archives/bj-pfd2/pfd-tutor/

**Comment by fupre** [comment]
赞！！！博主这个界面看着真舒服
